src/lib.rs

```diff
@@ -0,0 +1,90 @@
+/*
+ *   Copyright (c) 2020 
+ *   All rights reserved.
+ */
+
+pub mod read_create {
+    extern crate calamine;
+    use calamine::{open_workbook, Xlsx};
+    use std::fs::File;
+    use std::io::BufReader;
+
+
+pub fn create_file(input_file:&str) -> File{
+
+    let file_handle = File::create(input_file).expect("unable to create a output file");
+    return file_handle;
+
+} 
+
+pub fn open_file(output_file:&str) ->  Xlsx<BufReader<File>>  {
+
+    let workbook: Xlsx<_> = open_workbook(output_file).expect("unable to open the excel file");
```

Comment:
which excel file, remember while writing code u should think the user is fool he don't know anything so specify as much information as u can so that the fool user can understand